policyengine_us/parameters/gov/hhs/fpg.yaml

```diff
@@ -1,6 +1,7 @@
 description: Federal poverty guidelines published by the Department of Health and Human Services.
 first_person:
   CONTIGUOUS_US:
+    1992-01-01: 6_810
```

Comment:
Could you add the intervening years while you're here? This will avoid incorrect non-null values.